config/science_board_config.yaml

```diff
@@ -1,21 +1,26 @@
-led_map:

```

Comment:
ok so i guess now we just load in all this info into ros param server. what happens if two files use the same name? should we make the names more descriptive (e.g. info)

Reply:
I don't understand what you're asking here. You want to know what happens if two files use "led_map" ? Which name do you want to change to "info"?

Reply:
So like two different yaml files might interfere if they both use something as common as /info/

Reply:
So like info/sleep can easily be declared by a different configuration file

Reply:
RIght, yeah we could name it something more descriptive